stdlib/websub/src/main/ballerina/websub/h2_hub_persistence.bal

```diff
@@ -42,12 +42,14 @@ public type H2HubPersistenceStore object {
         self.subscriptionDbClient = subscriptionDbClient;
         var updateResult = self.subscriptionDbClient->update(CREATE_TOPICS_TABLE);
         if (updateResult is error) {
-            panic updateResult;
+            error e = updateResult;
+            panic e;
```

Comment:
We can merge L45-L46 right?
```suggestion
            panic updateResult;
```

Applicable elsewhere too?

Reply:
IIRC The type of updateResult inside type guard behaves as a union of errors(error|error) with the recent error type change. In order to panic the error, this casting was needed as the signature of particular function return error?.